tests/unit/test_results.py

```diff
@@ -118,7 +172,9 @@ def test_get_all_more_than_2k(self):
 
         # Assert that we performed a hits query and two search results queries
         self.assertEqual(len(self.cassette), 3)
-        self.assertEqual(len(granules), 2520)
+        self.assertEqual(
+            len(granules), int(self.cassette.responses[0]["headers"]["CMR-Hits"][0])
+        )
```

Comment:
@doug-newman-nasa, I just realized that we actually need this to be a combination of this plus what I suggested earlier. The issue with this assertion is that if this cassette is ever recreated at a point where the response would return a value for CMR-Hits > 3000, the test would then fail because it specifies count=3000. What we want is for the number of granules to match the min of 3000 and CMR-Hits:

```suggestion
        self.assertEqual(
            len(granules), min(3000, int(self.cassette.responses[0]["headers"]["CMR-Hits"][0]))
        )
```